### PR TITLE
Prod60 changes

### PR DESCRIPTION
**Pull request by sbesson**
Update `releases.csv` and `studies.csv` with the latest stats including the two new datasets.
Also uses `releases.csv` to fill the data footer (experimental, can be reverted if there are reasons to stick with the deployment-time JS)

To review check out https://snoopycrimecop.github.io/idr.openmicroscopy.org/about.html and http://snoopycrimecop.github.io/idr.openmicroscopy.org/studies/ and verify the metrics and links are correct.

### CODE REVIEW

File: _includes/foot-idr.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
                 <hr class="whitespace">
                 <div class="small-12 columns text-center">
                     <p class="version-number"><a href="{{ site.baseurl }}/index.html"><img id="version-number" src="{{ "/img/logos/logo-idr.svg" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="IDR logo"></a>
-                        version: <span id="version-number-display">{{ site.version }}</span>.
+                        version: {{ site.data.releases.last["Data release"] }}</span>.
```
[the remark this answers is not in the document]
Comment (sbesson) [reply _includes/foot-idr.html]: :+1: I'll fix it

**Comment by sbesson** [comment]
I agree with all the points raised above. Mid-term, I definitely would like to use these CSV files as the primary data sources and rationalizing the headers, formatting etc 

At the moment, however, I do not know we are in a position to deprecate the internal IDR Dataset map spreadsheet so these CSV files are effectively an export of this representation. Would you consider any of the points above as a blocker for `prod60` or should we capture each of them as RFEs and schedule them as iterations over the next releases?

**Comment by manics** [comment]
Not blockers, though would be nice to have consistent use of `,` separators or not

**Comment by sbesson** [comment]
After switching the tables to use the default formatting, the last commit re-exports the table as `tsv` and use the number value with the `round` operator. Using human-friendly sizes requires some more logic so probably outside the scope of this PR - see https://snoopycrimecop.github.io/idr.openmicroscopy.org/studies.html for the rendered output.

**Pull request merged**